### PR TITLE
Move getJtreg into getDependencies

### PR DESCRIPTION
**Pull request by annaibm**
-Moved the getJtreg version download logic into TKG/scripts/getDependencies.xml

resolves : https://github.com/adoptium/aqa-tests/issues/4848

### CODE REVIEW

**Comment by annaibm** [comment]
Grinder outputs:
openjdk: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36142/
Crypto test: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36150/
ssl-test:https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36151

File: scripts/getDependencies.xml [hunk 1]
```diff
@@ -22,6 +22,94 @@
...
+					<equals arg1="${JDK_IMPL}" arg2="openj9"/>
+				</not>
+			</and>
+		</condition>
+		<!-- Call the getJtreg target with the LIB parameter -->
+		<var name="LIB" unset="true"/>
+		<property name="LIB" value="${jtregTar}${openj9jtregtimeouthandler}"/>
+		<!-- <ant antfile="${TEST_ROOT}/TKG/scripts/getDependencies.xml" target="getJtreg">
+			<property name="LIB" value="${jtregTar}${openj9jtregtimeouthandler}"/>
+		</ant> -->
```
Comment (llxia) [inline scripts/getDependencies.xml:72]: remove the above lines

File: scripts/getDependencies.xml [hunk 2]
```diff
@@ -22,6 +22,94 @@
...
+			<condition>
+				<not>
+					<or>
+						<matches pattern="^([89]|1[02-6])$" string="${JDK_VERSION}"/>
+						<equals arg1="${JDK_VERSION}" arg2="11"/>
+						<matches pattern="^(17|2[12])$" string="${JDK_VERSION}"/>
+					</or>
+				</not>
+			</condition>
+		</fail>
```
Comment (llxia) [inline scripts/getDependencies.xml:48]: Please use the original logic https://github.com/adoptium/aqa-tests/blob/master/openjdk/build.xml#L44-L67

File: scripts/getDependencies.xml [hunk 3]
```diff
@@ -22,6 +22,94 @@
...
+		<condition property="openj9jtregtimeouthandler" value="">
+			<and>
+				<not>
+					<equals arg1="${JDK_IMPL}" arg2="ibm"/>
+				</not>
+				<not>
+					<equals arg1="${JDK_IMPL}" arg2="openj9"/>
+				</not>
+			</and>
+		</condition>
```
Comment (llxia) [inline scripts/getDependencies.xml:66]: Please use the original logic

File: scripts/getDependencies.xml [hunk 4]
```diff
@@ -22,6 +22,94 @@
...
+		</ant> -->
+	</target>
+
+    <target name="getJtreg" depends="getDependentLibs">
+		<mkdir dir="${DEST}"/>
+		<echo>LIB DIR is ${LIB_DIR}</echo>
+		<echo>Jtreg version is ${jtregTar}</echo>
+		<echo>jtregtimeouthandler version is ${openj9jtregtimeouthandler}</echo>
+		<echo>LIB  is ${LIB}</echo>
+		<echo>JDK_VERSION  is ${JDK_VERSION}</echo>
```
Comment (llxia) [inline scripts/getDependencies.xml:81]: Please remove the debugging messages

**Review by llxia — CHANGES_REQUESTED** [review]

**Comment by annaibm** [comment]
Made requested changes. Kindly review.
Grinder outputs:
openjdk: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36177/ (36178)
Crypto test: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36180/
ssl-test:https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36181

File: scripts/getDependencies.xml [hunk 5]
```diff
@@ -22,6 +22,81 @@
 
 	<!-- set default LIB property to all -->
 	<property name="LIB" value="all" />
+   <target name="getJtregVersion">
```
Comment (llxia) [inline scripts/getDependencies.xml:25]: Please fix indentation

**Comment by annaibm** [comment]
Grinder outputs:
openjdk: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36330/
openjdk with hotspot: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36373/
Crypto test: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36331/
ssl-test:https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36332/

File: scripts/getDependencies.xml [hunk 6]
```diff
@@ -22,6 +22,81 @@
...
+				<equals arg1="${JDK_IMPL}" arg2="openj9"/>
+			</or>
+			<then>
+				<property name="openj9jtregtimeouthandler" value=",tohandler_simple"/>
+			</then>
+			<else>
+				<property name="openj9jtregtimeouthandler" value=""/>
+			</else>
+		</if>
+		<!-- Call the getJtreg target with the LIB parameter -->
```
Comment (llxia) [inline scripts/getDependencies.xml:63]: Can we remove the comment?

Comment (annaibm) [reply scripts/getDependencies.xml:63]: Is it the comment on line 63?

Comment (llxia) [reply scripts/getDependencies.xml:63]: yes

**Review by llxia — APPROVED** [review]
LGTM

**Review by karianna — APPROVED** [review]

**Pull request merged**